python/paddle/v2/fluid/layers/nn.py

```diff
@@ -1262,7 +1262,7 @@ def conv2d(input,
         raise ValueError("use_cudnn should be True or False")
 
     input_shape = input.shape
-    filter_shape = [num_filters, num_filter_channels] + filter_size
+    filter_shape = [num_filters, num_filter_channels] + list(filter_size)
```

Comment:
Maybe add an "elif" after line 1256 to convert to list when it's a tuple? And throw a readable error for other unsupported types?

Reply:
thanks for your advice, I think we should define a `convert_to_list` function in `layers/utils.py`, just like what tensorflow does.